geometry_msgs/Transform pose
float64 dim_x
float64 dim_y
float64 dim_z